mintpy/smallbaselineApp.py

```diff
@@ -1063,8 +1065,72 @@ def plot_result(self, print_aux=True, plot=True):
         start_time = time.time()
         if self.template['mintpy.plot'] and plot:
             print('\n******************** plot & save to pic ********************')
-            print(self.plot_sh_cmd)
-            subprocess.Popen(self.plot_sh_cmd, shell=True).wait()
+
+            dem_file = os.path.join(self.workDir, 'inputs', 'geometryRadar.h5')
+            if not os.path.isfile(dem_file):
+                dem_file = os.path.join(self.workDir, 'inputs', 'geometryGeo.h5')
+            mask_file = os.path.join(self.workDir, 'maskTempCoh.h5')
+            pic_dir = os.path.join(self.workDir, 'pic')
+            ifgram_stack = os.path.join(self.workDir, 'inputs', 'ifgramStack.h5')
+            tropo_model = self.template['mintpy.troposphericDelay.weatherModel']
+            geo_dir = os.path.join(self.workDir, 'geo')
+
+            # view options
+            opt1 = ['--nodisplay', '--dpi', '150', '--update']
+            opt2 = ['--nodisplay', '--dpi', '150', '--update', '--mask', mask_file, '--noaxis', '-u', 'cm', '--wrap-range', '-10', '10']
+
+            iargs_list = [
+                ['velocity.h5'] + opt1,
+                ['temporalCoherence.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                ['maskTempCoh.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                [dem_file] + opt1,
+                [ifgram_stack] + opt1 + ['-unwrapPhase', '--zero-mask', '--wrap'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase', '--zero-mask'],
+                # [ifgram_stack] + opt1 + ['-coherence', '--mask', 'no'],
+                # [ifgram_stack] + opt1 + ['-connectComponent', '--mask', 'no'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase_bridging', '--zero-mask'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase_closure', '--zero-mask'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase_bridging_phaseClosure', '--zero-mask'],
+                ['avgPhaseVelocity.h5'] + opt1,
+                ['avgSpatialCoh.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                ['maskConnComp.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                ['timeseries.h5'] + opt2,
+                ['timeseries_LODcor.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF_demErr.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF_ramp.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF_ramp_demErr.h5'] + opt2,
+                [f'timeseries_{tropo_model}.h5'] + opt2,
+                [f'timeseries_{tropo_model}_demErr.h5'] + opt2,
+                [f'timeseries_{tropo_model}_ramp.h5'] + opt2,
+                [f'timeseries_{tropo_model}_ramp_demErr.h5'] + opt2,
+                [f'timeseries_ramp.h5'] + opt2,
+                [f'timeseries_demErr_ramp.h5'] + opt2,
+                [f'timeseries_demErr.h5'] + opt2,
```

Comment:
We should use the following two lines for all time-series files here.
```python
['timeseries.h5', opt2],
['timeseries_*.h5', opt2],
```

Reply:
Okay. I will fix this.